serverless/develop/serverless-applications.adoc

```diff
@@ -57,11 +57,12 @@ ifdef::openshift-enterprise[]
 // Using Knative services w/ restrictive NetworkPolicies
 include::modules/serverless-services-network-policies.adoc[leveloffset=+1]
 endif::[]
-// should maybe move to admin guide? asked serving team
+// move to admin guide, outside scope of this PR
```

Comment:
Enabling the feature looks to me an admin issue. Using the feature is part of the development/operation process of the application.